spec/decorators/user_decorator_spec.rb

```diff
@@ -105,10 +105,26 @@
     end
   end
 
+  describe "#config_font_name" do
+    it "replaces 'default' with font configured for the site in SiteConfig" do
+      expect(user.config_font).to eq("default")
```

Comment:
Shouldn't this be `serif`?

```suggestion
      expect(user.config_font).to eq("serif")
```

Reply:
User's `config_font` would still be `default` string in the DB (otherwise we would have to update all the existing users column in the DB).

When requesting the `config_font_name` from the decorator it should render their font of choice or swap out `default` for the chosen default font set by the instance admin. In the second case (user still has `default`) the tests checks against all possible fonts down in following lines of this same spec.

Still in doubt about whether we should allow all fonts though ([question here](https://github.com/forem/forem/pull/9650#issuecomment-673519126))

Reply:
> Still in doubt about whether we should allow all fonts though

Nah, I think we should only allow the following:

- serif
- sans-serif
- dyslexia one

Once we have better theming infrastructure we could let ppl choose their own default fonts, but for now we should limit the choices to the three fonts above.

Reply:
Done! These will be the fonts available to choose from in `/admin/config`:

<img width="1162" alt="Screen Shot 2020-08-13 at 10 56 00" src="https://user-images.githubusercontent.com/6045239/90163682-a5171980-dd53-11ea-9c8e-7c4628747974.png">

I did found out that comments are not using the selected font... Maybe this is because of the changes in the CSS? Can you take a look and see why is this hapenning @ludwiczakpawel? I'm not sure I follow very well what changed that affected this

Reply:
@fdoxyz yea, i skipped it intentionally since I'm working on Comments section anyway, so it will be tackled in a separate PR.